migrations/versions/1956c5d25718_.py

```diff
@@ -0,0 +1,40 @@
+"""empty message
+
+Revision ID: 1956c5d25718
+Revises: 161275d54fe0
+Create Date: 2024-04-30 09:43:02.707801
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '1956c5d25718'
+down_revision = '161275d54fe0'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    with op.batch_alter_table('iati_activity', schema=None) as batch_op:
+        batch_op.create_index(batch_op.f('ix_iati_activity_reporting_organisation'), ['reporting_organisation'], unique=False)
+
+    with op.batch_alter_table('iati_line', schema=None) as batch_op:
+        batch_op.create_foreign_key(None, 'receiver_organisation_type', ['receiver_organisation_type'], ['code'], ondelete='CASCADE')
+        batch_op.create_foreign_key(None, 'provider_organisation_type', ['provider_organisation_type'], ['code'], ondelete='CASCADE')
```

Comment:
I think it would be good to give the foreign key a name, like in here (and also to apply this to the downgrade function):
https://github.com/iati-data-access/data-backend/blob/main/migrations/versions/75a6e15cb36b_.py#L38